java/org/contikios/cooja/serialsocket/SerialSocketServer.java

```diff
@@ -242,7 +242,7 @@ public Dimension getPreferredSize() {
         try {
           listenPortField.commitEdit();
         } catch (ParseException ex) {
-          logger.error(ex);
+          logger.error("Listen port is not a number");
```

Comment:
What do you think about including the current value in the error message using `listenPortField.getText()`?